src/application/services/useNoteUtils.ts

```diff
@@ -0,0 +1,72 @@
+import { type OutputData } from '@editorjs/editorjs';
+
+/**
+ * Get the title of the note
+ *
+ * @param content - content of the note
+ * @returns the title of the note
+ */
+export const getTitle = (content: OutputData): string => {
+  const limitCharsForNoteTitle = 50;
+  const firstNoteBlock = content.blocks[0];
+
+  /**
+   *  If the heading is empty, return 'Untitled'
+   */
+  if (firstNoteBlock.data.text == '') {
+    return 'Untitled';
+  } else {
+    return firstNoteBlock.data.text.slice(0, limitCharsForNoteTitle);
+  }
+};
+
+/**
+ *
+ * @param updatedAt - the date the note was last updated
+ * @returns the last updated time in a human-readable format
+ */
+export const getUpdateTime = (updatedAt: string): string => {
```

Comment:
Let's move logic of working with dates to separate service

Reply:
Also I would consider using some library to work with dates easier. [Day.js](https://day.js.org/) for example. But this is a topic for separate pull request